dtfs-central-api/src/types/fee-records.ts

```diff
@@ -34,7 +34,7 @@ export type KeyingSheetRow = {
   facilityId: string;
   exporter: string;
   feePayments: {
-    dateReceived: Date;
+    dateReceived: Date | undefined;
```

Comment:
Just because this is something that gets sent in a response body, could we have it be `Date | null`?